### ISSUE TITLE
Remote access using `cheta` not `Ska.engarchive` fails

### ISSUE THREAD

**Issue by taldcroft** [sot/eng_archive#192]
tl;dr: I think this might be resolved by using native namespace packages (https://github.com/sot/skare3/issues/290)

```
C:\Users\jkristoff>python
Python 3.6.2 |Anaconda, Inc.| (default, Sep 30 2017, 11:52:29) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.

>>> import os
>>> os.environ['SKA_ACCESS_REMOTELY'] = 'True'
>>> from cheta import fetch_eng as fetch

Enter hostname (or IP) of Ska server (enter to cancel): ...
Enter your login username [jkristoff]:
Enter your password:
Establishing connection to  ...
Loading MSID names from Ska eng archive server...

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ska.engarchive-4.47.3-py3.6.egg\cheta\fetch_eng.py", line 4, in <module>
    from .fetch import *  # noqa
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ska.engarchive-4.47.3-py3.6.egg\cheta\fetch.py", line 296, in <module>
    all_colnames = load_msid_names(all_msid_names_files)
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ska.engarchive-4.47.3-py3.6.egg\cheta\fetch.py", line 227, in wrapper
    return remote_access.execute_remotely(func, *args, **kwargs)
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ska.engarchive-4.47.3-py3.6.egg\cheta\remote_access.py", line 169, in execute_remotely
    return dview.apply_sync(fcn, *args, **kwargs)
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\view.py", line 226, in apply_sync
    return self._really_apply(f, args, kwargs, block=True)
  File "<decorator-gen-131>", line 2, in _really_apply
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\view.py", line 52, in sync_results
    ret = f(self, *args, **kwargs)
  File "<decorator-gen-130>", line 2, in _really_apply
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\view.py", line 37, in save_ids
    ret = f(self, *args, **kwargs)
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\view.py", line 576, in _really_apply
    return ar.get()
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\asyncresult.py", line 169, in get
    raise self.exception()
  File "C:\Users\jkristoff\Documents\MATLAB\FOT_Tools\Python\python3_Windows_64bit\lib\site-packages\ipyparallel\client\asyncresult.py", line 226, in _resolve_result
    raise r
ipyparallel.error.RemoteError: ModuleNotFoundError(No module named 'cheta.fetch')
```

I think the problem is related to the fact that `cheta` is being installed as a package within the `Ska.engarchive` namespace package and that this is not playing well with the IPython parallel machinery that figures out the the correct remote imports in order to replicate the current environment and allow a remote function call to work properly.

### Locally
```
>>> import cheta
>>> import pprint
>>> pprint.pprint(cheta.__dict__)
...
'__cached__': '/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/Ska.engarchive-4.47.4-py3.6.egg/cheta/__pycache__/__init__.cpython-36.pyc',
 '__doc__': None,
 '__file__': '/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/Ska.engarchive-4.47.4-py3.6.egg/cheta/__init__.py',
 '__loader__': <_frozen_importlib_external.SourceFileLoader object at 0x7fef70011ba8>,
 '__name__': 'cheta',
 '__package__': 'cheta',
 '__path__': ['/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/Ska.engarchive-4.47.4-py3.6.egg/cheta'],
 '__spec__': ModuleSpec(name='cheta', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7fef70011ba8>, origin='/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/Ska.engarchive-4.47.4-py3.6.egg/cheta/__init__.py', submodule_search_locations=['/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/Ska.engarchive-4.47.4-py3.6.egg/cheta']),
 '__version__': '4.47.4',
 'ska_helpers': <module 'ska_helpers' from '/Users/aldcroft/miniconda3/envs/ska3/lib/python3.6/site-packages/ska_helpers/__init__.py'>,
 'test': <function test at 0x7fef70018f28>}
```
### Remotely
```
In [1]:     import os
   ...:     os.environ.pop('SKA', None)
   ...:     os.environ.pop('ENG_ARCHIVE', None)
   ...:     os.environ['SKA_ACCESS_REMOTELY'] = 'True'
   ...:     from cheta import remote_access
   ...: 

In [2]: def get_stuff():
   ...:     import cheta
   ...:     return cheta.__dict__

In [4]: remote_access.establish_connection()
Enter hostname (or IP) of Ska server (enter to cancel):131.142.113.4
Enter your login username [aldcroft]:taldcroft
Enter your password:
Establishing connection to 131.142.113.4...
Out[4]: True

In [5]: remote_access.execute_remotely(get_stuff)
Out[5]: 
{'__doc__': None,
 '__loader__': <_frozen_importlib_external._NamespaceLoader at 0x7f96a0860e80>,
 '__name__': 'cheta',
 '__package__': 'cheta',
 '__path__': _NamespacePath(['/proj/sot/ska3/flight/lib/python3.6/site-packages/Ska.engarchive-4.47.3-py3.6.egg/cheta']),
 '__spec__': ModuleSpec(name='cheta', loader=None, origin='namespace', submodule_search_locations=_NamespacePath(['/proj/sot/ska3/flight/lib/python3.6/site-packages/Ska.engarchive-4.47.3-py3.6.egg/cheta']))}
```
Note in the above that there is no `__file__` or `__version__` or `test`, meaning the package import was incomplete.  This might be related to the new SCM versioning.

**Comment by taldcroft** [comment]
This appears to be fixed now since it is possible to use `cheta` for the import for remote access on both Mac and Windows. See e.g. the thread on Slack starting with "Probably a rudimentary question", where @matthewdahmer does the following on Windows:
```
In [3]: from cheta import fetch
In [4]: fptemp = fetch.Msid('fptemp_11', '2020:001', '2021:090')
Enter hostname (or IP) of Ska server (enter to cancel):131.142.xxx.yyy
Enter your login username [mdahmer]:mdahmer
Enter your password:
Establishing connection to 131.142.xxx.yyy
Loading MSID names from Ska eng archive server...
Getting interval data from DB on Ska eng archive server...
Getting time data from Ska eng archive server...
Getting msid data for FPTEMP_11 from Ska eng archive server...
```
